packages/xo-web/src/common/intl/messages.js

```diff
@@ -570,7 +570,8 @@ const messages = {
   newSrCreate: 'Create',
   newSrNamePlaceHolder: 'Storage name',
   newSrDescPlaceHolder: 'Storage description',
-  newSrAddressPlaceHolder: 'Address',
+  newSrAddressPlaceHolder: 'IP Address',
```

Comment:
Address can be a host name or a IP. You can let it "Address" it's clear.

Reply:
I don't think `Address` provides any meaningful info, maybe `93.184.216.34 or nfs.example.net`.